sample/src/NimblePros.SampleToDo.Core/ProjectAggregate/ToDoItemId.cs

```diff
@@ -3,4 +3,4 @@ namespace NimblePros.SampleToDo.Core.ProjectAggregate;
 
 [ValueObject<int>]
 [Instance("New", 0)]
-public partial struct ToDoItemId; 
+public partial struct ToDoItemId;
```

Comment:
Optimised slightly by removing the space 👍 (actually, I changed this when I was investigating by removing the `Instance` attribute - but it had no effect on the outcome of the investigation)

Reply:
Does your generator care about whitespace?